api/web/templates/web/project_view.html

```diff
@@ -81,27 +81,28 @@ <h3 class="mb-0">
                               </h3>
                             </div>
                             <div id="collapseOne" class="collapse" aria-labelledby="headingOne" data-parent="#accordion">
-                              <div class="card-body" id="chartId" style="width:100%; height:400px;"></div>
+                              <div class="card-body" id="chartId" style="width:100%; height:400px; text-transform: capitalize;"></div>
                             </div>
                           </div>
 
                           <div class="card">
                             <div class="card-header bg-gradient-light" id="headingTwo">
                               <h3 class="mb-0">
-                                <button class="btn btn-link collapsed text-dark" data-toggle="collapse" data-target="#collapseTwo" aria-expanded="false" aria-controls="collapseTwo">
+                                <button class="btn btn-link collapsed text-dark" onclick="hot.render()" data-toggle="collapse" data-target="#collapseTwo" aria-expanded="false" aria-controls="collapseTwo">
```

Comment:
A handsontable não tá renderizando corretamente ao abrir a aba pois o html da tabela não está disponível no momento do click. A maneira ideal para contornar esse problema é adicionar um listener no evento "shown.bs.collapse" do bootstrap. Quando o conteúdo do collapse for completamente renderizado, esse evento é disparado.

Na parte do código do javascript eu adicionei a sugestão do código do listener junto com um spinner para mostrar ao usuário que o código está sendo carregado.

```suggestion
                                <button class="btn btn-link collapsed text-dark" data-toggle="collapse" data-target="#collapseTwo" aria-expanded="false" aria-controls="collapseTwo">
```